### ISSUE TITLE
Add a Merge Tag modifier to the Entry moderation Merge Tags to output the URL only

### ISSUE THREAD

**Issue by rafaehlers** [GravityKit/GravityView#1986]
There's no way at this moment to just output the URL for the [entry moderation merge tags](https://docs.gravitykit.com/article/904-entry-moderation-merge-tags).

This will be useful for so many use cases, including generating a QRCode, as requested by a user.

Related: https://github.com/GravityKit/GravityView/blob/develop/includes/class-gravityview-entry-approval-merge-tags.php#L227

**Comment by floatytheastronaut** [comment]
FreeScout #77233. <br>https://support.gravitykit.com/conversation/77233?folder_id=1

**Comment by mrcasual** [comment]
Implemented in fa45dce5f.

**Comment by rafaehlers** [comment]
@mrcasual This never worked:

On a form notification:
![image](https://github.com/user-attachments/assets/ac783d4a-7ec9-45c4-a631-6b4d044ad337)

![image](https://github.com/user-attachments/assets/7b73fa56-4e81-4117-a5c8-69ede7b8fd93)

Inside a custom content field:
![image](https://github.com/user-attachments/assets/22ee091f-1a4b-4563-888f-ff5a000303b9)

![image](https://github.com/user-attachments/assets/7e410231-b44c-406a-9778-644ea068ab21)

Tested on GV 2.27.1

**Comment by mrcasual** [comment]
@rafaehlers, the issue never asked to add a new (`:url`) merge tag modifier. Your related issue pointed to the code where the merge tag can be modifier through a filter, and that's what was done here as well.

```
add_filter( 'gk/gravityview/entry/approval-link/params', function ( $params ) {
	$params['format'] = 'text';

	return $params;
} );
```

It works. If you want a merge tag modifier, then please create an a new issue.

**Comment by mrcasual** [comment]
@rafaehlers, implemented by 5565cca14. Please test.

**Comment by rafaehlers** [comment]
It works! Customer notified ✅
